eng/pipelines/templates/steps/npm-release-task.yml

```diff
@@ -0,0 +1,56 @@
+parameters:
+  Registry: ''
+
+steps:
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+  - ${{if eq(parameters.Registry, 'https://registry.npmjs.org/')}}:
+      - task: PowerShell@2
+        displayName: Get package list
+        inputs:
+          targetType: filePath
+          filePath: eng/tools/publish-to-npm.ps1
+          arguments: -pathToArtifacts $(Package.Archive) -accessLevel "public" -tag "$(Tag)" -additionalTag "$(AdditionalTag)" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)
```

Comment:
Where is Tag & AdditionalTag coming from?

Reply:
those were set in earlier steps in archetype-js-release  https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/stages/archetype-js-release.yml#L87-L88

Reply:
When we make this a reusable template we should probably take those as input parameters instead of magically expecting these variables to be set.